Task_20/YndexTest.cs

```diff
@@ -0,0 +1,25 @@
+using NUnit.Framework;
+using Yandex.Pages;
+
+namespace Yandex
+{
+    [TestFixture]
+    public class YndexTest: BaseTest
+    {
+        const string USER_NAME = "WebDriverCSharp";
+        const string PASSWORD = "Selenium";
+
+        [TestCase(USER_NAME, PASSWORD)]
+        public void SignInTest(string userName, string password)
+        {
+            HomePage homePage = new HomePage(driver);
+            homePage.GoToURl();
+            homePage.ClickSignInBtn();
+
+            SignInPage signInPage = new SignInPage(driver);
+            signInPage.SignIn(userName, password);
+
+            Assert.IsTrue(homePage.IsUserLoggedIn(USER_NAME));
```

Comment:
Please, add the second parameter here to extend description of reasons why this test could fail at this line. For example, $"Login operation failed under username {USER_NAME}"

Reply:
done